scripts/noshake.json

```diff
@@ -367,6 +367,7 @@
   "Mathlib.Lean.Expr.ExtraRecognizers": ["Mathlib.Data.Set.Operations"],
   "Mathlib.Lean.Expr.Basic": ["Batteries.Logic"],
   "Mathlib.GroupTheory.MonoidLocalization.Basic": ["Mathlib.Init.Data.Prod"],
+  "Mathlib.GroupTheory.Congruence.Defs": ["Mathlib.Tactic.FastInstance"],
```

Comment:
If you add `FastInstance` in the first block, the `IgnoreImport` group, then you do not have to individually unshake it elsewhere.

Reply:
Great! I just did the mindless thing and would happy to see something better done.